sdk/Cargo.toml

```diff
@@ -121,36 +118,58 @@ serde_with = "3.11.0"
 serde-transcode = "1.1.1"
 sha1 = "0.10.6"
 sha2 = "0.10.6"
-tempfile = "3.10.1"
+tempfile = "3.14"
 thiserror = "1.0.61"
 treeline = "0.1.0"
-url = "2.5.2"
-uuid = { version = "1.10.0", features = ["serde", "v4", "js"] }
+url = "2.5.3"
```

Comment:
FYI we can't take the 2.5.3 version of `url` until we've had Legal review a new license introduced by that version.